src/SprykerSdk/Integrator/Builder/Visitor/AddClassToClassListVisitor.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Builder\Visitor;
+
+use PhpParser\BuilderFactory;
+use PhpParser\Node;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\NodeVisitorAbstract;
+use SprykerSdk\Integrator\Helper\ClassHelper;
+
+class AddClassToClassListVisitor extends NodeVisitorAbstract
+{
+    /**
+     * @var string
+     */
+    protected $methodName;
+
+    /**
+     * @var string
+     */
+    protected $className;
+
+    /**
+     * @var bool
+     */
+    protected $methodFound = false;
+
+    /**
+     * @var string
+     */
+    protected $constantName;
+
+    /**
+     * @param string $methodName
+     * @param string $className
+     * @param string $constantName
+     */
+    public function __construct(string $methodName, string $className, string $constantName)
+    {
+        $this->methodName = $methodName;
+        $this->className = ltrim($className, '\\');
+        $this->constantName = $constantName;
+    }
+
+    /**
+     * @param \PhpParser\Node $node
+     *
+     * @return \PhpParser\Node|int|null
+     */
+    public function enterNode(Node $node)
+    {
+        if ($node->getType() === 'Stmt_ClassMethod' && $node->name->toString() === $this->methodName) {
```

Comment:
I would move this strings to the constant. It's pretty much possible that if someday this library updated, this class names can be changed